workers/ohsome_quality_analyst/base/indicator.py

```diff
@@ -41,19 +38,28 @@ class Result:
         timestamp_oqt (datetime): Timestamp of the creation of the indicator
         timestamp_osm (datetime): Timestamp of the used OSM data
             (e.g. Latest timestamp of the ohsome API results)
-        label (str): Traffic lights like quality label
-        value (float): The result value as float ([0, 1])
-        description (str): Description of the result
+        label (str): Traffic lights like quality label: Green, yellow or red. The value
+            is determined by the result classes
+        value (float): The result value
+        class_ (int): The result class. An integer between 1 and 5. It maps to the
+            result labels. This value is used by the reports to determine an overall
+            result.
+        description (str): The result description.
         svg (str): Figure of the result as SVG
     """
 
-    timestamp_oqt: datetime
-    timestamp_osm: Optional[datetime]
-    label: Literal["green", "yellow", "red", "undefined"]
-    value: Optional[float]
-    description: str
-    svg: str
-    html: str
+    timestamp_oqt: datetime = datetime.now(timezone.utc)  # UTC datetime object
+    timestamp_osm: Optional[datetime] = None
+    value: Optional[float] = None
+    class_: Optional[Literal[1, 2, 3, 4, 5]] = None
+    description: str = ""
+    svg: str = ""
```

Comment:
Since `description` and `svg` are both set in the `__init__` function, I would not set them with default values here.

Reply:
Changed to no default